lib/stdlib/test/lists_SUITE.erl

```diff
@@ -2783,3 +2786,15 @@ do_error_info(L0) ->
     NYI = [{F,lists:duplicate(A, '*'),nyi} || {F,A} <- Bifs -- Tests],
     L = lists:sort(NYI ++ L1),
     error_info_lib:test_error_info(lists, L, [snifs_only]).
+
+uniq_1(_Config) ->
+    [] = lists:uniq([]),
+    [foo] = lists:uniq([foo]),
+    ["foo", "bar", "zoo"] = lists:uniq(["foo", "foo", "bar", "foo", "zoo", "foo", "bar", "zoo"]),
+    [a, 1, b, 2] = lists:uniq([a, a, a, 1, b, 2, a, 2, 1]),
+    [<<"home">>, "home"] = lists:uniq([<<"home">>, "home"]),
+    [3.14159, 2.71828, 3.17] = lists:uniq([3.14159, 3.14159, 2.71828, 3.17]).
```

Comment:
```suggestion
    ["foo", "bar", "zoo"] = lists:uniq(["foo", "foo", "bar", "foo", "zoo",
                                        "foo", "bar", "zoo"]),
    [a, 1, b, 2] = lists:uniq([a, a, a, 1, b, 2, a, 2, 1]),
    [<<"home">>, "home"] = lists:uniq([<<"home">>, "home"]),
    [3.14159, 2.71828, 3.17] = lists:uniq([3.14159, 3.14159, 2.71828, 3.17]),
    [42, 42.0] = lists:uniq([42, 42.0, 42, 42.0]),
    ok.
```

In the suggestion, I have:

* Broken a long line.
* Added a test for equal integer and float values.
* Ended the test case with `ok` since this value shows up in the test case log and we don't want arbitrary values there.